client/components/LevelOne.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LevelOne = (props) => {
+  const {
+    nextTwoWeeks, times, max, next, change, count, date, time,
+  } = props;
+
+  const guestCount = [];
+  for (let j = 1; j <= 12; j += 1) {
+    if (j <= max) {
+      guestCount.push(j);
+    } else {
+      break;
+    }
+  }
+
+  return (
+    <div>
+      <h3>
+        Make a Reservation
+      </h3>
+      <div>
+        <select onChange={change} name="selectedDate" value={date}>
+          {nextTwoWeeks.map(e => (
```

Comment:
Assuming that 'e' represents a date, you might call 'e' something like 'date'.

Reply:
In any other context, I'd say descriptive variables are best. However for me, a descriptive variable in this context is more confusing because you're name a variable only used on these lines of code and isn't part of the bigger story. If I called it 'possibleDate', and I was looking at it at a glance, I'd say, what is possible date? Oh, its an element of 'nextTwoWeeks'.  Why make the extra step of mental logic?

That said, I'll change it.

Sadly, date is used by something else.